src/main/java/com/coveros/selenified/services/check/Check.java

```diff
@@ -0,0 +1,181 @@
+     *
+     * @param expectedkeys - a list with string keys expected in the json
+     *                     response
+     * @param actualValue  - the JsonObject that is being checked for jsonvalues
+     * @return boolean - does the provided jsonObject contain all of the key value pairs
+     */
+    public boolean doesJsonObjectExcludeKeys(List<String> expectedkeys, JsonObject actualValue) {
+        boolean pass = true;
+        for (String key : expectedkeys) {
+            if (actualValue != null && actualValue.has(key)) {
+                pass = false;
+            }
+        }
+        return pass;
+    }
+
+    /**
+     * Checks whether the provided jsonObject contains each of the provided key value pairs
+     *
+     * @param expectedPairs - a hashmap with string key value pairs expected in the json
+     *                      response
+     * @param actualValue   - the JsonObject that is being checked for jsonvalues
+     * @return boolean - does the provided jsonObject contain all of the key value pairs
+     */
+    public boolean doesJsonObjectContainPairs(Map<String, Object> expectedPairs, JsonObject actualValue) {
+        boolean pass = (actualValue != null);
+        for (Map.Entry<String, Object> entry : expectedPairs.entrySet()) {
+            if (actualValue != null && actualValue.has(entry.getKey())) {
+                Object objectVal = castObject(entry.getValue(), actualValue.get(entry.getKey()));
+                if (!entry.getValue().equals(objectVal)) {
+                    pass = false;
+                }
+            } else {
+                pass = false;
+            }
+        }
+        return pass;
+    }
+
+    /**
+     * Checks whether the provided jsonObject contains each of the provided key value pairs
+     *
+     * @param expectedPairs - a hashmap with string key value pairs expected in the json
+     *                      response
+     * @param actualValue   - the JsonObject that is being checked for jsonvalues
+     * @return boolean - does the provided jsonObject contain all of the key value pairs
+     */
+    public boolean doesJsonObjectExcludePairs(Map<String, Object> expectedPairs, JsonObject actualValue) {
+        boolean pass = true;
+        for (Map.Entry<String, Object> entry : expectedPairs.entrySet()) {
+            if (actualValue != null && actualValue.has(entry.getKey()) && entry.getValue().equals(castObject(entry.getValue(), actualValue.get(entry.getKey())))) {
+                pass = false;
+            }
+        }
+        return pass;
+    }
+
+    boolean recordResult(String expected, String actual, boolean success) {
+        if (success) {
+            this.reporter.pass("", expected, actual);
+        } else {
+            this.reporter.fail("", expected, actual);
+        }
+        return success;
+    }
+
+    //TODO needs matching, does not contain - consider breaking this into assertEquals, assertConains, assertExcludes (to match web)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1135)